systemverilog-plugin/UhdmAst.cc

```diff
@@ -3527,6 +3528,9 @@ void UhdmAst::process_logic_typespec()
     }
     visit_one_to_many({vpiRange}, obj_h, [&](AST::AstNode *node) { packed_ranges.push_back(node); });
     add_multirange_wire(current_node, packed_ranges, unpacked_ranges);
+    if (vpi_get(vpiSigned, obj_h) == true) {
+        current_node->is_signed = true;
+    }
```

Comment:
Do we only want to set ``is_signed`` when it is ``true`` (what about the case, when ``current_node->is_signed`` is ``true`` and ``vpi_get(vpiSigned, obj_h)`` is ``false``)? Maybe something like this would be better:
```suggestion
    current_node->is_signed = vpi_get(vpiSigned, obj_h);
```

Reply:
`is_signed` is always initialized to `false`, so we shouldn't run into this case. I agree that always setting the status is clearer though, as we check it anyway. Changed.